shell/browser/api/electron_api_session.h

```diff
@@ -55,6 +56,29 @@ class ElectronBrowserContext;
 
 namespace api {
 
+class WebContents;
+
+#if BUILDFLAG(ENABLE_ELECTRON_EXTENSIONS)
+class ExtensionTabDetails {
+ public:
+  ExtensionTabDetails();
+  ExtensionTabDetails(const ExtensionTabDetails& other);
+  ~ExtensionTabDetails();
+
+  int window_id = 0;
+  int index = -1;
+  int group_id = -1;
+  int opener_tab_id = 0;
+  bool active = false;
+  bool highlighted = false;
+  bool pinned = false;
+  bool discarded = false;
+  bool auto_discardable = false;
+  std::string muted_reason = "";
+  std::string muted_extension_id = "";
```

Comment:
There is no need to assign "" to std::string, the default constructor of std::string already does it.